Example/UIViewController-DisplayInDrawer/ContentViewController.swift

```diff
@@ -81,13 +109,17 @@ extension ContentViewController: DrawerConfiguration {
     }
 
     func middlePositionY(for parentHeight: CGFloat) -> CGFloat? {
-        guard isViewLoaded else { return nil }
-        return parentHeight - separatorView.frame.minY
+        guard isViewLoaded, useMiddlePosition else { return nil }
+        return halfOpenPositionY(for: parentHeight)
```

Comment:
So is it halfOpen or middle position ? Keep the naming consistent please

Reply:
The trouble here is, that in the context of this particular content controller its `halfOpenPosition` can be in lib naming either middle (in case we have three positions) or bottom (if we have two positions). The confusion comes from the fact, that I try to demonstrate two drawer configurations on a single content controller, which is not likely to happen in real world.